packages/graphql-server/src/functions/graphql.test.ts

```diff
@@ -2,15 +2,15 @@
 /* eslint-disable @typescript-eslint/ban-ts-comment */
 import { envelop, Plugin } from '@envelop/core'
 
-import { context, getPerRequestContext } from '@redwoodjs/api'
+import { context, getPerRequestContext } from '@redwoodjs/graphql-server'
 
-import { useUserContext, useRedwoodGlobalContextSetter } from './graphql'
+import { usePopulateContext, useRedwoodGlobalContextSetter } from './graphql'
 
 const createContextHandler = (userContext?: any) => {
```

Comment:
Do you think we could type the userContext argument here? If I understand correctly its `() => Record<string, any>`

Reply:
Done.